tools/generators/workspace-generator/schema.ts

```diff
@@ -0,0 +1,10 @@
+export interface WorkspaceGeneratorGeneratorSchema {
```

Comment:
is this generated from the `schema.json` or maintained manually ?

Reply:
generated. we can add that tooling later.